### PR TITLE
feat: add ruby mysql2/rails chinese tutorial

### PR DESCRIPTION
**Pull request by hooopo**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/14642
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: develop/dev-guide-sample-application-ruby-mysql2.md [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+7. 保存 `.env` 文件。
+
+</div>
+<div label="TiDB Dedicated">
+
+1. 导航到 [**Clusters**](https://tidbcloud.com/console/clusters) 页面，然后点击你的目标集群的名称，进入其概览页面。
+
+2. 点击右上角的 **Connect**。会显示一个连接对话框。
+
+3. 点击 **Allow Access from Anywhere**，然后点击 **Download TiDB cluster CA** 以下载 CA 证书。
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-mysql2.md:115]:
```suggestion
3. 点击 **Allow Access from Anywhere**，然后点击 **Download TiDB cluster CA** 以下载 CA 证书。
```
```suggestion
3. 在对话框中点击 **Allow Access from Anywhere**，然后点击 **Download TiDB cluster CA** 下载 TiDB Cloud 提供的 CA 证书。
```

File: develop/dev-guide-sample-application-ruby-mysql2.md [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+
+   关于如何获取连接字符串的更多细节，请参考 [TiDB Dedicated standard connection](https://docs.pingcap.com/tidbcloud/connect-via-standard-connection)。
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-mysql2.md:117]:
```suggestion
   更多配置细节，可参考 [TiDB Dedicated 标准连接教程（英文）](https://docs.pingcap.com/tidbcloud/connect-via-standard-connection)。

File: develop/dev-guide-sample-application-ruby-mysql2.md [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+
+4. 运行以下命令复制 `.env.example` 并将其重命名为 `.env`：
+
+    ```shell
+    cp .env.example .env
+    ```
+
+5. 编辑 `.env` 文件，按照以下方式设置环境变量，并将连接对话框中的连接参数替换为相应的占位符 `<>`：
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-mysql2.md:125]:
```suggestion
5. 编辑 `.env` 文件，按照以下方式设置环境变量，并将占位符 `<>`替换为连接对话框中相应的连接参数：
```

File: develop/dev-guide-sample-application-ruby-mysql2.md [hunk 4]
```diff
@@ -0,0 +1,302 @@
...
+    DATABASE_NAME=test
+    DATABASE_ENABLE_SSL=true
+    DATABASE_SSL_CA=<downloaded_ssl_ca_path>
+    ```
+
+   > **注意：**
+   >
+   > 当使用 Public Endpoint 连接到 TiDB Dedicated 集群时，建议启用 TLS 连接。
+   >
+   > 要启用 TLS 连接，将 `DATABASE_ENABLE_SSL` 修改为 `true`，并使用 `DATABASE_SSL_CA` 指定从连接对话框下载的 CA 证书的文件路径。
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-mysql2.md:141]:
```suggestion
   > 要启用 TLS 连接，请将 `DATABASE_ENABLE_SSL` 修改为 `true`，并使用 `DATABASE_SSL_CA` 指定从连接对话框下载的 CA 证书的文件路径。
```

File: develop/dev-guide-sample-application-ruby-mysql2.md [hunk 5]
```diff
@@ -0,0 +1,302 @@
...
+</div>
+<div label="本地部署的 TiDB">
+
+1. 运行以下命令复制 `.env.example` 并将其重命名为 `.env`：
+
+    ```shell
+    cp .env.example .env
+    ```
+
+2. 编辑 `.env` 文件，按照以下方式设置环境变量，并将连接对话框中的连接参数替换为相应的占位符 `<>`：
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-mysql2.md:154]:
```suggestion
2. 编辑 `.env` 文件，按照以下方式设置环境变量，并将占位符 `<>`替换为你的 TiDB 集群的连接参数：

File: develop/dev-guide-sample-application-ruby-mysql2.md [hunk 6]
```diff
@@ -0,0 +1,302 @@
...
+
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-mysql2.md:155]: 这里的英文也需要修改，需要删除 “replace the corresponding placeholders `<>` with connection parameters in the connection dialog:” 中的 “in the connection dialog”

File: develop/dev-guide-sample-application-ruby-mysql2.md [hunk 7]
```diff
@@ -0,0 +1,302 @@
...
+    ```
+
+   如果你在本地运行 TiDB，那么默认的主机地址是 `127.0.0.1`，密码为空。
+
+3. 保存 `.env` 文件。
+
+</div>
+</SimpleTab>
+
+### 步骤 4：运行代码并检查结果
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-mysql2.md:171]:
```suggestion
### 第 4 步：运行代码并查看结果
```

File: develop/dev-guide-sample-application-ruby-mysql2.md [hunk 8]
```diff
@@ -0,0 +1,302 @@
...
+
+运行以下命令来执行示例代码：
+
+```shell
+ruby app.rb
+```
+
+如果连接成功，控制台将输出 TiDB 集群的版本，如下所示：
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-mysql2.md:179]:
```suggestion
如果连接成功，你的终端将会输出所连接集群的版本信息：
```

File: develop/dev-guide-sample-application-ruby-mysql2.md [hunk 9]
```diff
@@ -0,0 +1,302 @@
...
+ℹ️ Got Player 12: Player { id: 12, coins: 100, goods: 100 }
+🔢 Added 50 coins and 50 goods to player 12, updated 1 row.
+🚮 Deleted 1 player data.
+```
+
+## 示例代码片段
+
+你可以参考以下示例代码片段来完成你自己的应用程序开发。
+
+要获取完整的示例代码以及如何运行它，请查看 [tidb-samples/tidb-ruby-mysql2-quickstart](https://github.com/tidb-samples/tidb-ruby-mysql2-quickstart) 仓库。
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-mysql2.md:196]:
```suggestion
完整代码及其运行方式，见代码仓库 [tidb-samples/tidb-ruby-mysql2-quickstart](https://github.com/tidb-samples/tidb-ruby-mysql2-quickstart)。
```

File: develop/dev-guide-sample-application-ruby-mysql2.md [hunk 10]
```diff
@@ -0,0 +1,302 @@
...
+
+🆕 Created a new player with ID 12.
+ℹ️ Got Player 12: Player { id: 12, coins: 100, goods: 100 }
+🔢 Added 50 coins and 50 goods to player 12, updated 1 row.
+🚮 Deleted 1 player data.
+```
+
+## 示例代码片段
+
+你可以参考以下示例代码片段来完成你自己的应用程序开发。
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-mysql2.md:194]:
```suggestion
你可参考以下关键代码片段，完成自己的应用开发。
```

File: develop/dev-guide-sample-application-ruby-mysql2.md [hunk 11]
```diff
@@ -0,0 +1,302 @@
...
+
+要获取完整的示例代码以及如何运行它，请查看 [tidb-samples/tidb-ruby-mysql2-quickstart](https://github.com/tidb-samples/tidb-ruby-mysql2-quickstart) 仓库。
+
+### 使用连接选项连接到 TiDB
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-mysql2.md:198]:
```suggestion
### ### 连接到 TiDB
```

File: develop/dev-guide-sample-application-ruby-mysql2.md [hunk 12]
```diff
@@ -0,0 +1,302 @@
...
+
+以下代码使用环境变量中定义的选项建立到 TiDB 的连接：
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-mysql2.md:200]:
```suggestion
下面的代码使用环境变量中定义的连接选项来建立与 TiDB 集群的连接。
```

File: develop/dev-guide-sample-application-ruby-mysql2.md [hunk 13]
```diff
@@ -0,0 +1,302 @@
...
+
+```ruby
+require 'dotenv/load'
+require 'mysql2'
+Dotenv.load # Load the environment variables from the .env file
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-mysql2.md:205]:
```suggestion
Dotenv.load # 从 .env 文件中加载环境变量
```

File: develop/dev-guide-sample-application-ruby-mysql2.md [hunk 14]
```diff
@@ -0,0 +1,302 @@
...
+  database: ENV['DATABASE_NAME'] || 'test'
+}
+options.merge(ssl_mode: :verify_identity) unless ENV['DATABASE_ENABLE_SSL'] == 'false'
+options.merge(sslca: ENV['DATABASE_SSL_CA']) if ENV['DATABASE_SSL_CA']
+client = Mysql2::Client.new(options)
+```
+
+> **注意：**
+>
+> 对于 TiDB Serverless，当使用 Public Endpoint 时，必须通过 `DATABASE_ENABLE_SSL` 启用 TLS 连接，但是你**不必**通过 `DATABASE_SSL_CA` 指定 SSL CA 证书，因为 mysql2 gem 会按照特定的顺序搜索现有的 CA 证书，直到找到一个文件。
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-mysql2.md:221]:
```suggestion
> 对于 TiDB Serverless，当使用 Public Endpoint 时，**必须**通过 `DATABASE_ENABLE_SSL` 启用 TLS 连接，但是你**不需要**通过 `DATABASE_SSL_CA` 指定 SSL CA 证书，因为 mysql2 gem 会按照特定的顺序搜索现有的 CA 证书，直到找到相应的文件。
```

File: develop/dev-guide-sample-application-ruby-mysql2.md [hunk 15]
```diff
@@ -0,0 +1,302 @@
...
+
+### 插入数据
+
+以下查询创建一个具有两个字段的玩家，并返回 `last_insert_id`：
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-mysql2.md:225]:
```suggestion
以下查询创建一个具有两个字段的 player，并返回 `last_insert_id`：
```

File: develop/dev-guide-sample-application-ruby-mysql2.md [hunk 16]
```diff
@@ -0,0 +1,302 @@
...
+```ruby
+def create_player(client, coins, goods)
+  result = client.query(
+    "INSERT INTO players (coins, goods) VALUES (#{coins}, #{goods});"
+  )
+  client.last_id
+end
+```
+
+有关更多信息，请参考 [插入数据](/develop/dev-guide-insert-data.md)。
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-mysql2.md:236]:
```suggestion
更多信息，请参考[插入数据](/develop/dev-guide-insert-data.md)。
```

File: develop/dev-guide-sample-application-ruby-mysql2.md [hunk 17]
```diff
@@ -0,0 +1,302 @@
...
+
+### 查询数据
+
+以下查询通过 ID 返回特定玩家的记录：
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-mysql2.md:240]:
```suggestion
以下查询通过 ID 返回特定 player 的记录：
```

File: develop/dev-guide-sample-application-ruby-mysql2.md [hunk 18]
```diff
@@ -0,0 +1,302 @@
...
+```ruby
+def get_player_by_id(client, id)
+  result = client.query(
+    "SELECT id, coins, goods FROM players WHERE id = #{id};"
+  )
+  result.first
+end
+```
+
+有关更多信息，请参考 [查询数据](/develop/dev-guide-get-data-from-single-table.md)。
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-mysql2.md:251]:
```suggestion
更多信息，请参考[查询数据](/develop/dev-guide-get-data-from-single-table.md)。
```

File: develop/dev-guide-sample-application-ruby-mysql2.md [hunk 19]
```diff
@@ -0,0 +1,302 @@
...
+
+### 更新数据
+
+以下查询通过 ID 更新特定玩家的记录：
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-mysql2.md:255]:
```suggestion
以下查询通过 ID 更新特定 player 的记录：
```

File: develop/dev-guide-sample-application-ruby-mysql2.md [hunk 20]
```diff
@@ -0,0 +1,302 @@
...
+```ruby
+def update_player(client, player_id, inc_coins, inc_goods)
+  result = client.query(
+    "UPDATE players SET coins = coins + #{inc_coins}, goods = goods + #{inc_goods} WHERE id = #{player_id};"
+  )
+  client.affected_rows
+end
+```
+
+有关更多信息，请参考 [更新数据](/develop/dev-guide-update-data.md)。
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-mysql2.md:266]:
```suggestion
更多信息，请参考[更新数据](/develop/dev-guide-update-data.md)。
```

File: develop/dev-guide-sample-application-ruby-mysql2.md [hunk 21]
```diff
@@ -0,0 +1,302 @@
...
+
+### 删除数据
+
+以下查询删除特定玩家的记录：
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-mysql2.md:270]:
```suggestion
以下查询删除特定 player 的记录：
```

File: develop/dev-guide-sample-application-ruby-mysql2.md [hunk 22]
```diff
@@ -0,0 +1,302 @@
...
+```ruby
+def delete_player_by_id(client, id)
+  result = client.query(
+    "DELETE FROM players WHERE id = #{id};"
+  )
+  client.affected_rows
+end
+```
+
+有关更多信息，请参考 [删除数据](/develop/dev-guide-delete-data.md)。
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-mysql2.md:281]:
```suggestion
更多信息，请参考[删除数据](/develop/dev-guide-delete-data.md)。
```

File: develop/dev-guide-sample-application-ruby-mysql2.md [hunk 23]
```diff
@@ -0,0 +1,302 @@
...
+
+## 最佳实践
+
+默认情况下，mysql2 gem 可以按照特定的顺序搜索现有的 CA 证书，直到找到一个文件。
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-mysql2.md:285]:
```suggestion
默认情况下，mysql2 gem 可以按照特定的顺序搜索现有的 CA 证书，直到找到相应的文件。
```

File: develop/dev-guide-sample-application-ruby-mysql2.md [hunk 24]
```diff
@@ -0,0 +1,302 @@
...
+
+1. 对于 Debian、Ubuntu、Gentoo、Arch 或 Slackware，路径为 `/etc/ssl/certs/ca-certificates.crt`
+2. 对于 RedHat、Fedora、CentOS、Mageia、Vercel 或 Netlify，路径为 `/etc/pki/tls/certs/ca-bundle.crt`
+3. 对于 OpenSUSE，路径为 `/etc/ssl/ca-bundle.pem`
+4. 对于 macOS 或 Alpine（docker 容器），路径为 `/etc/ssl/cert.pem`
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-mysql2.md:290]:
```suggestion
1. 对于 Debian、Ubuntu、Gentoo、Arch 或 Slackware，路径为 `/etc/ssl/certs/ca-certificates.crt`。
2. 对于 RedHat、Fedora、CentOS、Mageia、Vercel 或 Netlify，路径为 `/etc/pki/tls/certs/ca-bundle.crt`。
3. 对于 OpenSUSE，路径为 `/etc/ssl/ca-bundle.pem`。
4. 对于 macOS 或 Alpine（docker 容器），路径为 `/etc/ssl/cert.pem`。
```

File: develop/dev-guide-sample-application-ruby-mysql2.md [hunk 25]
```diff
@@ -0,0 +1,302 @@
...
+
+虽然可以手动指定 CA 证书路径，但在多环境部署场景中，这可能会造成很大的不便，因为不同的机器和环境可能会在不同的位置存储 CA 证书。因此，建议将 `sslca` 设置为 `nil`，以便在不同环境中灵活且方便地部署。
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-mysql2.md:292]:
```suggestion
尽管可以手动指定 CA 证书路径，但在多环境部署场景中这可能会引起不必要的麻烦，因为不同的机器和环境可能存储 CA 证书的位置不同。因此，建议将 `sslca` 设置为 `nil`，方便在不同环境中灵活且方便地部署。
```

File: develop/dev-guide-sample-application-ruby-mysql2.md [hunk 26]
```diff
@@ -0,0 +1,302 @@
...
+
+## 下一步
+
+- 从 [mysql2 的文档](https://github.com/brianmario/mysql2#readme)中了解更多关于 mysql2 驱动的使用情况。
+- 通过 [开发者指南](/develop/dev-guide-overview.md) 中的章节，了解 TiDB 应用开发的最佳实践，例如：[插入数据](/develop/dev-guide-insert-data.md)、[更新数据](/develop/dev-guide-update-data.md)、[删除数据](/develop/dev-guide-delete-data.md)、[查询数据](/develop/dev-guide-get-data-from-single-table.md)、[事务](/develop/dev-guide-transaction-overview.md) 和 [SQL 性能优化](/develop/dev-guide-optimize-sql-overview.md)。
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-mysql2.md:297]:
```suggestion
- 你可以继续阅读开发者文档的其它章节来获取更多 TiDB 应用开发的最佳实践。例如：[插入数据](/develop/dev-guide-insert-data.md)，[更新数据](/develop/dev-guide-update-data.md)，[删除数据](/develop/dev-guide-delete-data.md)，[单表读取](/develop/dev-guide-get-data-from-single-table.md)，[事务](/develop/dev-guide-transaction-overview.md)，[SQL 性能优化](/develop/dev-guide-optimize-sql-overview.md)等。
```

File: develop/dev-guide-sample-application-ruby-mysql2.md [hunk 27]
```diff
@@ -0,0 +1,302 @@
...
+- 通过专业的 [TiDB 开发者课程](https://www.pingcap.com/education/) 学习，并在通过考试后获得 [TiDB 认证](https://www.pingcap.com/education/certification/)。
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-mysql2.md:298]:
```suggestion
- 如果你更倾向于参与课程进行学习，我们也提供专业的 [TiDB 开发者课程](https://cn.pingcap.com/courses-catalog/back-end-developer/?utm_source=docs-cn-dev-guide)支持，并在考试后提供相应的[资格认证](https://learn.pingcap.com/learner/certification-center)。
```

File: develop/dev-guide-sample-application-ruby-rails.md [hunk 28]
```diff
@@ -0,0 +1,277 @@
...
+  <<: *default
+  database: quickstart_test
+
+production:
+  <<: *default
+```
+
+> **注意**
+>
+> 对于 TiDB Serverless，当使用 Public Endpoint 时，**必须**通过在 `DATABASE_URL` 中设置 `ssl_mode` 查询参数为 `verify_identity` 来启用 TLS 连接，但是你**不需要**通过 `DATABASE_URL` 指定 SSL CA 证书，因为 mysql2 gem 会按照特定的顺序搜索现有的 CA 证书，直到找到一个文件。
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-rails.md:216]:
```suggestion
> 对于 TiDB Serverless，当使用 Public Endpoint 时，**必须**通过在 `DATABASE_URL` 中设置 `ssl_mode` 查询参数为 `verify_identity` 来启用 TLS 连接，但是你**不需要**通过 `DATABASE_URL` 指定 SSL CA 证书，因为 mysql2 gem 会按照特定的顺序搜索现有的 CA 证书，直到找到相应的文件。
```

File: develop/dev-guide-sample-application-ruby-rails.md [hunk 29]
```diff
@@ -0,0 +1,277 @@
...
+更多信息，请参考[删除数据](/develop/dev-guide-delete-data.md)。
+
+## 最佳实践
+
+默认情况下，mysql2 gem 可以按照特定的顺序搜索现有的 CA 证书，直到找到相应的文件。
+
+1. 对于 Debian、Ubuntu、Gentoo、Arch 或 Slackware，路径为 `/etc/ssl/certs/ca-certificates.crt`。
+2. 对于 RedHat、Fedora、CentOS、Mageia、Vercel 或 Netlify，路径为 `/etc/pki/tls/certs/ca-bundle.crt`。
+3. 对于 OpenSUSE，路径为 `/etc/ssl/ca-bundle.pem`。
+4. 对于 macOS 或 Alpine（docker 容器），路径为 `/etc/ssl/cert.pem`。
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-rails.md:265]:
```suggestion
1. 对于 Debian、Ubuntu、Gentoo、Arch 或 Slackware，证书的默认存储路径为 `/etc/ssl/certs/ca-certificates.crt`。
2. 对于 RedHat、Fedora、CentOS、Mageia、Vercel 或 Netlify，证书的默认存储路径为 `/etc/pki/tls/certs/ca-bundle.crt`。
3. 对于 OpenSUSE，证书的默认存储路径为 `/etc/ssl/ca-bundle.pem`。
4. 对于 macOS 或 Alpine（docker 容器），证书的默认存储路径为 `/etc/ssl/cert.pem`。
```

File: develop/dev-guide-sample-application-ruby-rails.md [hunk 30]
```diff
@@ -0,0 +1,277 @@
...
+
+## 下一步
+
+- 从 [ActiveRecord 文档](https://guides.rubyonrails.org/active_record_basics.html)中了解更多关于 ActiveRecord ORM 的用法。
+- 你可以继续阅读开发者文档的其它章节来获取更多 TiDB 应用开发的最佳实践。例如：[插入数据](/develop/dev-guide-insert-data.md)，[更新数据](/develop/dev-guide-update-data.md)，[删除数据](/develop/dev-guide-delete-data.md)，[单表读取](/develop/dev-guide-get-data-from-single-table.md)，[事务](/develop/dev-guide-transaction-overview.md)，[SQL 性能优化](/develop/dev-guide-optimize-sql-overview.md)等。
+- 如果你更倾向于参与课程进行学习，我们也提供专业的 [TiDB 开发者课程](https://cn.pingcap.com/courses-catalog/back-end-developer/?utm_source=docs-cn-dev-guide)支持，并在考试后提供相应的[资格认证](https://learn.pingcap.com/learner/certification-center)。
+
+## 需要帮助？
+
+在 [Discord](https://discord.gg/vYU9h56kAX) 频道上提问。
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-rails.md:277]: 中文文档，这里是否需要替换为 Ask TUG

File: develop/dev-guide-sample-application-ruby-rails.md [hunk 31]
```diff
@@ -0,0 +1,277 @@
...
+
+## 示例代码片段
+
+你可参考以下关键代码片段，完成自己的应用开发。
+
+完整代码及其运行方式，见代码仓库 [tidb-samples/tidb-ruby-rails-quickstart](https://github.com/tidb-samples/tidb-ruby-rails-quickstart)。
+
+### 连接到 TiDB
+
+`config/database.yml` 中的以下代码使用 `DATABASE_URL` 配置连接到 TiDB：
```
Comment (qiancai) [inline develop/dev-guide-sample-application-ruby-rails.md:194]:
```suggestion
`config/database.yml` 中的以下代码使用 `DATABASE_URL` 系统变量的配置连接到 TiDB：
```

File: develop/dev-guide-sample-application-ruby-mysql2.md (hunk 6 above)
Comment (hooopo) [reply develop/dev-guide-sample-application-ruby-mysql2.md:155]: https://github.com/pingcap/docs/pull/14822

**Review by qiancai — APPROVED** [review]

**Pull request merged**